docs/DEPLOYMENT.md

```diff
@@ -17,6 +17,9 @@ We push the latest translation to staging branch, via poEditor console.
 
 ### Update app version and changelog.md
 
-Run `yarn version` to enter the new app version and update the `CHANGELOG.md` file.
+1. Update app version:
+    Run `yarn version` to enter the new app version
```

Comment:
The few times I ran this I had to do `yarn version minor | major | patch` because its a yarn command and then `yarn run version` to execute the script inside the package.json file.

Reply:
I usually just do it manually. Will update it . THanks.